torch/_torch_docs.py

```diff
@@ -3234,6 +3234,53 @@ def merge_dicts(*dicts):
     tensor([ 0.,  2.,  1.,  0.])
 """.format(**common_args))
 
+add_docstr(torch.histogram,
+           r"""
+histogram(input, bins=10, weights=None, range=None, density=False) -> (Tensor, Tensor)
+histogram(input, bins, weights=None, density=False) -> (Tensor, Tensor)
+
+Computes the histogram of a tensor.
+
+Args:
+    {input}
+    bins (int or 1D Tensor): If int, defines the number of bins. If Tensor,
+        defines a monotonically increasing sequence of bin edges, allowing
+        for non-uniform bins. Default: 10.
+    range (tuple of floats, optional): Defines the lower and upper ends of the
```

Comment:
Just add a sentence here about the Default behavior (that the min and max value in the tensor are used for the range).